bin/gitlab-api-v3

```diff
@@ -103,42 +103,18 @@ try {
 
     exit 0 if !defined $data;
 
-    # Replace all non HASH/ARRAY references with strings.  This is
-    # implemented generically, but really only causes JSON boolean
-    # object to be stringified which makes sense for non-JSON serializers.
-    if ($format ne 'JSON') {
-        $data = clean_data( $data );
```

Comment:
This change not only allow non-deserialized data through, but also removes the clean_data bit.

Could you explain why clean_data isn't needed anymore?  It was a while ago when I wrote this, so I'm a bit fuzzy on the details.